gradle/publish.gradle

```diff
@@ -9,7 +9,7 @@ bintray {
     user = 'szczepiq'
     key = System.env.MOCKITO_BINTRAY_API_KEY
 
-    publications = ['mockitoCore']
+    publications = ['mockitoCore', 'mockitoAndroid']
```

Comment:
I don't think this works. Did you test it? 'mockitoAndroid' is a publication in a different Gradle subproject.